apps/src/lib/script-editor/FlexGroup.jsx

```diff
@@ -148,10 +148,10 @@ class FlexGroup extends Component {
 
     return (
       <div>
-        {_.keys(groups).map((group, groupIndex) => (
+        {_.keys(groups).map(group => (
```

Comment:
should we rename this file/component to `<FlexCategory/>`, or does `<FlexGroup/>` still make more sense here?

Reply:
Good idea! I'd like to defer this suggestion for now and tackle it as part of a larger renaming after other PRs in this chain of PRs have landed, but now could be a good time to decide on the new name.

One thing to note is that the `FlexGroup` component is really the set of all flex categories, so I don't think `FlexCategory` would be a good name but `FlexCategories` or `LessonGroups` might be. `StageEditor`or `LessonsEditor` (similar to `ScriptEditor`) might be good names too since this class also has the responsibility of serializing levels and stages. Let me know if any of those sound good or bad to you!

Reply:
i think waiting to rename and pluralizing whatever terminology we decide on sounds perfect 👍